docs/guides/on-demand.mdx

```diff
@@ -342,60 +342,16 @@ const response = await axios
 
 ### Response
```

Comment:
I can't comment on the `Request` above since it wasn't changed here, but I think it has a few errors and is over-complicated:
 - It **must not** instruct users to provide an `Authorization` header on the request. It will cause confusion on developers as if they had to call this from their backend and thus proxy all uploads for their users, when in fact the upload API has no authorization. The URL itself is pre-signed to be used once for that specific asset. Should be called directly from the browser with no auth.
 - It **should not** instruct users to provide a `Content-Type`. That's completely ignored, no reason to complicate the request.
 - URL argument is malformed, the code doesn't work if you copy paste it somewhere.
 - Indentation is weird, gets confusing to understand.
 - Since you're providing a function and not a runnable script, it makes sense to receive function arguments for anything not specified in your snippet.
 - Code is written for `node.js`, not the browser (uses `fs` instead of `File`)

Fixing all but the last point, would have it be like this:
```js
const uploadAsset = async (url, path) => {
  const response = await axios.put(url, { data: fs.createReadStream(path) });
  console.log(JSON.stringify(response.data));
};
```

Reply:
Ok, will make these adjustments

Reply:
We should still update the code to work in the browser instead of node though.